### DISCUSSION TITLE
Image Rendering After Next.js Deployment

### DISCUSSION THREAD

**Discussion by nesbita** [vercel/vercel#6573] [Help]
Hello, 
My images were rendering properly prior to deploying my Next.js project with Vercel, but now one of them won't load.  It just has the file name and an image icon.  I selected the option to load the image and it does nothing, and if I select to open the image in another tab I get a 404 error.  Any ideas on how to get my image to show again?
Thanks

**Comment by mcsdevv** [comment]
Hi @nesbita, sorry to hear this! Can you share with us the deployment and the specific path the image isn't loading on?

**Reply by nesbita** [reply]
Hi, 
Thanks for getting back to me.  The deployment is here https://vercel.com/nesbita and the code for the image is attached.  If it matters, I used z-index while styling to make it a background image.  
![Vercel Screenshot](https://user-images.githubusercontent.com/82684778/129097470-91ac5d9b-c611-49e0-b951-fe1426623cde.png)

**Reply by leerob** [reply]
Background image example here:
- https://image-component.nextjs.gallery/background
- https://github.com/vercel/next.js/blob/canary/examples/image-component/pages/background.js
